File: launchdarkly/ruby/src/GetIntegrationDeliveryConfigurationByIdExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::IntegrationDeliveryConfigurationsBetaApi.new.get_integration_delivery_configuration_by_id(
        nil, # project_key
        nil, # environment_key
        nil, # integration_key
        nil, # id
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling IntegrationDeliveryConfigurationsBetaApi#get_integration_delivery_configuration_by_id: #{e}"
end

File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/GetIntegrationDeliveryConfigurationByIdExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class GetIntegrationDeliveryConfigurationByIdExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            var response = new IntegrationDeliveryConfigurationsBetaApi(config).GetIntegrationDeliveryConfigurationById(
                projectKey: null,
                environmentKey: null,
                integrationKey: null,
                id: null
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling IntegrationDeliveryConfigurationsBetaApi#GetIntegrationDeliveryConfigurationById: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
